nservicebus/azure-service-bus/multiple-namespaces-support.md

```diff
@@ -0,0 +1,63 @@
+---
+title: Multiple namespace support
+summary: Configuring Azure ServiceBus transport to support different partitioning strategies. 
+component: ASB
+tags:
+- Cloud
+- Azure
+- Transports 
+reviewed: 2016-04-18
+---
+
+`Azure Service Bus` transport enables different _namespace partitioning strategies_ to cover different scenarios like High Availability and multiple Data Center support or to overtake [service limits](https://azure.microsoft.com/en-us/documentation/articles/service-bus-quotas/).  
```

Comment:
> transport enables different _namespace partitioning strategies_ to cover different scenarios like

Perhaps:
"transport supports different _namespace partitioning strategies_ to cover different scenarios such as"